src/modules/bootloader/main.py

```diff
@@ -637,42 +637,61 @@ def run_grub_install(fw_type, partitions, efi_directory):
                                    "--bootloader-id=" + efi_bootloader_id,
                                    "--force"])
     else:
-        assert efi_directory is None
-        if libcalamares.globalstorage.value("bootLoader") is None:
-            return
+        if efi_directory is not None:
```

Comment:
Perhaps combine these? `if efi-directory-set but not install-hybrid: print(error); return;`, I think it would be good to log some kind of error anyway.

Reply:
Good idea. Implemented a warning message with `libcalamares.utils.warning`.